Changelog.md

```diff
@@ -59,6 +59,7 @@
 * Elements with ARIA roles must ensure required owned elements are present (`src/audits/RequiredOwnedAriaRoleMissing.js`)
 * Avoid positive integer values for tabIndex (`src/audits/TabIndexGreaterThanZero.js`)
 * This element has an unsupported ARIA attribute (`src/audits/UnsupportedAriaAttribute.js`)
+* A tabpanel should be related to a tab via aria-controls or aria-labelledby (`src/audits/UncontrolledTabpanel.js)
```

Comment:
As with #218, we'll need to move this Changelog entry up above the latest release line (sorry!)